docs_theme/installation.md

```diff
@@ -1,2 +1,5 @@
-Instructions about installation here
+```    npm install jsthermalcomfort```
 
+if you want to use jsthermalcomfort package without installing it on your local machine, you can import with:
```

Comment:
also mention that you can import it in the web directly using the method below, and mention that for it to work you need to mark it as module:
```html
<script type="module">
  import { models, utilities, pschymetrics } from "https://cdn.jsdelivr.net/gh/FedericoTartarini/jsthermalcomfort/lib/esm/index.js"
</script>
```